inugami_webapp/src/main/webapp/js/app/components/forms/input.time.slots.ts

```diff
@@ -0,0 +1,233 @@
+import { Component, OnInit, forwardRef, Input, Output, EventEmitter } from '@angular/core';
+import { NG_VALUE_ACCESSOR, ControlValueAccessor } from '@angular/forms';
+
+
+export const INPUT_TIME_SLOTS_VALUE_ACCESSOR: any = {
+    provide: NG_VALUE_ACCESSOR,
+    useExisting: forwardRef(() => InputTimeSlots),
+    multi: true
+}; 
+
+@Component({
+    selector: 'input-time-slots',
+    providers: [INPUT_TIME_SLOTS_VALUE_ACCESSOR],
+    template: `
+    <div [ngClass]="{'inugami-input-time-slots': true}">
```

Comment:
You template is a little bit too big, can you externalize this one into a html file ?